org.eclipse.draw2d/src/org/eclipse/draw2d/graph/CompoundBreakCycles.java

```diff
@@ -29,38 +29,34 @@ class CompoundBreakCycles extends GraphVisitor {
 	 * from the list.
 	 */
 	private NodeList graphNodes;
-	private NodeList sL = new NodeList();
+	private final NodeList sL = new NodeList();
 
-	private boolean allFlagged(NodeList nodes) {
-		for (int i = 0; i < nodes.size(); i++) {
-			if (nodes.getNode(i).flag == false)
-				return false;
-		}
-		return true;
+	private static boolean allFlagged(NodeList nodes) {
+		return nodes.stream().noneMatch(n -> !n.flag);
```

Comment:
I think it'd be more readable to use `allMatch(n -> n.flag)`, to avoid the double-negation

Reply:
Changed.